hack/build/docker/cdi-func-test-file-host-http/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:28
```

Comment:
I'm see either f28 or f27 in the current round of prs. Should we standardize on one?

Reply:
Yes.  We need a small PR that adds a base fedora image , then the Dockerfiles for all containers should be changed to `FROM base`.   This would reduce the os version to 1 place.